examples/orthographic.js

```diff
@@ -0,0 +1,92 @@
+/* global itowns, document, renderer */
+// # Orthographic viewer
+
+// Define geographic extent: CRS, min/max X, min/max Y
+const extent = new itowns.Extent(
+    'EPSG:3857',
+    -20026376.39, 20026376.39,
+    -20048966.10, 20048966.10);
+
+// `viewerDiv` will contain iTowns' rendering area (`<canvas>`)
+const viewerDiv = document.getElementById('viewerDiv');
+
+
+const r = viewerDiv.clientWidth / viewerDiv.clientHeight;
+const camera = new itowns.THREE.OrthographicCamera(
+    extent.west(), extent.east(),
+    extent.east() / r, extent.west() / r,
+    0, 1000);
+
+// Instanciate PlanarView
+var view = new itowns.PlanarView(viewerDiv, extent, { renderer, maxSubdivisionLevel: 10, camera });
+view.tileLayer.disableSkirt = true;
```

Comment:
I think this `disableSkirt` is not obvious to the newcomers. Can you add a comment to explain why it's needed?